analyses/molecular-subtyping-pathology/03-incorporate-pathology-feedback.Rmd

```diff
@@ -526,6 +526,68 @@ compiled_df <- compiled_df %>%
   )
 ```
 
+
+### Chordoma samples pathology review 
+As discussed in [ticket](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/608) we will assign subtypes to chordoma samples through pathology review only since molecular subtypes were not sufficient.
+	
+ sample_id | molecular_subtype
+-- | --
+7316-1101 |	CHDM, poorly differentiated
+7316-2935 |	CHDM, poorly differentiated
+7316-3295 |	CHDM, poorly differentiated
+7316-4062 |	CHDM, poorly differentiated
+7316-921 |	CHDM, conventional
+7316-3632	| CHDM, conventional
+
+```{r}
+compiled_df %>%
+  filter(sample_id %in% c("7316-1101",
+                       "7316-2935",
+                       "7316-3295",
+                       "7316-4062",
+                       "7316-921",
+                       "7316-3632")) 
+
+```
+
+
+```{r}
+
+compiled_df <- compiled_df %>%
+    mutate(molecular_subtype = case_when(
+      sample_id %in% c("7316-1101",
+                       "7316-2935",
+                       "7316-3295",
+                       "7316-4062") ~ "CHDM, poorly differentiated",
+      sample_id %in% c("7316-921",
+                       "7316-3632") ~ "CHDM, conventional",
+      TRUE ~ molecular_subtype
+    ),
+    integrated_diagnosis = case_when(
```

Comment:
It looks like `molecular_subtype` and `integrated_diagnosis` are identical? If we are always expecting it to be this way, could just make `integrated_diagnosis = molecular_subtype` so its simpler.

Reply:
Hmm molecular-subtyping values are `CHDM, poorly differentiated or CHDM, conventional `and intergrated_diagnosis values should be `"Chordoma, poorly differentiated" or "Chordoma, conventional"` so there are slight changes in each column.

Reply:
@kgaonkar6 you can use this to shorten the code:
```
mutate(integrated_diagnosis = str_replace(molecular_subtype, "CHDM, ", "Chordoma, "))
```